packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -521,6 +543,101 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
         );
     }
 
+    onCtxMenu(event: React.MouseEvent<HTMLDivElement>): void {
+        event.preventDefault();
+        event.stopPropagation();
+        const refNode = this.treeRef.current;
+        let calculatedX = 0;
+        let calculatedY = 0;
+        if (refNode) {
+            calculatedX = event.clientX;
+            calculatedY = event.clientY - refNode.getBoundingClientRect().top;
+        }
+
+        const { show } = useContextMenu({
+            id: MENU_ID + this.props.outputDescriptor.id
+        });
+
+        show(event, {
+            props: {},
+            position: { x: calculatedX, y: calculatedY }
+        });
+    }
+
+    protected handleItemClick = (params: ItemParams): void => {
+        const props = {
+            outputDescriptor: this.props.outputDescriptor,
+            selectedRow: this.state.selectedRow,
```

Comment:
`selectedRow` and `selectedRows` are to close in name and name should be clearer. Maybe rename new variable and releated methods/signals to make it clearer.

Reply:
Ok - i will change it in the next patch!